circleci.sh

```diff
@@ -89,7 +89,7 @@ coverage() {
 
     # rebuild dmd with coverage enabled
     # use the just build dmd as host compiler this time
-    mv src/dmd src/host_dmd
+    mv generated/dmd src/host_dmd
```

Comment:
Yes, copying (`cp --link`) the build dmd binary and dmd.conf to `src/` would help transitioning a lot. There are really lots of scripts and it's only 2 files.